opal/models.py

```diff
@@ -916,8 +952,29 @@ def get_display_name(cls):
         return cls._meta.verbose_name
 
     @classmethod
+    def _get_template(cls, template, prefixes=None):
+        result = []
```

Comment:
Bikeshedding perhaps but... this isn't the `result` - I expect you to return it when I see this...